v1/tester/runner.go

```diff
@@ -308,6 +308,13 @@ func (r *Runner) RaiseBuiltinErrors(enabled bool) *Runner {
 	return r
 }
 
+// StrictBuiltinErrors causes builtin evaluation errors to be reported
+// as test ERRORs instead of FAILs.
+func (r *Runner) StrictBuiltinErrors(enabled bool) *Runner {
```

Comment:
Since we don't call this anywhere else than in a test, is it your intention that this fix is only applicable to API usages? If so, then the claim to be fixing #6816 - which explicitly mentions this as an issue when running from the commandline - is at least partially invalid.
Also, that issue does not mention strict mode, which seems to be the target here.